### PR TITLE
Swtich to Github Actions

### PR DESCRIPTION
**Pull request by dianabarsan**
Maven publishing still TBD.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+on: [push, pull_request]
+
+jobs:
+  js-bootstrap:
+    name: Test JS bootstrap on Node v${{ matrix.node-version }}
+    runs-on: ubuntu-18.04
+
+    strategy:
+      matrix:
+        node-version: [ 8.x, 10.x, 12.x ]
```
Comment (garethbowen) [inline .github/workflows/test.yml:12]: Does it work on 14 and 16? If so we should test those too.

**Review by garethbowen — CHANGES_REQUESTED** [review]
This makes me happy.

Don't forget to delete the travis.yml!

File: .github/workflows/test.yml (hunk 1 above)
Comment (dianabarsan) [reply .github/workflows/test.yml]: Switched matrix to test with 12, 14, 16.

**Comment by dianabarsan** [comment]
> Don't forget to delete the travis.yml!

I wanted to keep it around until I make sure everything that it contains still works. Do you mind?

**Pull request merged**